docs/Creating-Your-Own-EmulationStation-Theme.md

```diff
@@ -1,166 +1,153 @@
-  - [Download the file](https://drive.google.com/file/d/0B2TMeZ6iEFvHYkNrMGVQSldkZ0U/view?usp=sharing)
-- Install/unzip it where you want it. I have it on F:/ Drive, so the path is F:\emulationstation
-- Download [this](https://dl.dropboxusercontent.com/u/14838442/fake-games.zip) zip file. Within it are 3 folders: 'gb', 'nes' and 'snes'. Within each folder are 15 blank .txt files, each named after a game on that system. These files have also had their extensions changed to something that ES will see as a rom.
-  - **These are not real roms. They are blank text files that have been renamed to look like roms.**
-  - Emulationstation sees these as real games. You won't be able to play them (since they are really just blank text files), but they will scrape with the built in Emulationstation scraper. They are also extremely small, so you don't have to worry about your test-bed getting too big.
-- Copy those folders into F:\emulationstation\\.emulationstation\roms
+- Download the portable EmulationStation by @herb_fargus, see [references](#references)
+- Install/unzip it where you want it. For example if you have it on `F:\` drive, so the path is `F:\emulationstation`
+- Download the fake games' collection, see [references](#references). Within it are three folders: `gb`, `nes` and `snes`. Within each folder are 15 empty files, each named after a game on that system. These files have also had their extensions changed to something that EmulationStation will recognize as a ROM.
+  - _These are not real ROMs. They are blank text files that have been renamed to look like ROMs._
+  - EmulationStation sees these as real games. You won't be able to play them (since they are really just empty files), but they will scrape with the built-in EmulationStation scraper. They are also tiny, so you don't have to worry about your test-bed getting too big.
+- Copy those folders into `F:\emulationstation\\.emulationstation\roms`
+
 
+EmulationStation should now be set up with 45 (fake) games on three Systems, and ready for you to use.
 
-Emulationstation should now be set up with 45 (fake) games on 3 Systems, and ready for you to use.
+Double click `Launch Portable (Windowed).bat` to start EmulationStation.
 
-Double click `Launch Portable (Windowed).bat` to start Emulationstation.
+### Use Samba Share of EmulationStation's Themes on RetroPie
 
----
+You may also edit theme files directly on the Raspberry. However, you should use a wired network connection between your desktop and the Raspberry to enable smooth editing. Open `\\retropie\configs\all\themes` in the Windows File Explorer. You should be able to see (and edit) the currently installed themes of your EmulationStation installation. If you do not see any theme, then install the Spare theme as outlined below. For reloading a theme (see [shortcuts](#keyboard-shortcuts-in-emulationstation)) you should also have a keyboard attached to your RetroPie, and it should be in reach from your desktop system.
```

Comment:
The share path is `\\retropie\configs\all\emulationstation\themes` - apologies for suggesting an incomplete path.

Reply:
No worries. I also could have verified this before editing/commiting. Ok now.